packages/ra-ui-materialui/src/detail/Edit.spec.tsx

```diff
@@ -885,7 +919,9 @@ describe('<Edit />', () => {
                     i18nProvider={i18nProvider}
                 >
                     <ResourceDefinitionContextProvider
-                        definitions={{ foo: { recordRepresentation: 'title' } }}
+                        definitions={{
+                            foo: { recordRepresentation: 'title' },
```

Comment:
```suggestion
                            foo: { name: 'foo', recordRepresentation: 'title' },
```

I believe TS is angry about this